### PR TITLE
Add CopyObject Function #128

### PR DESCRIPTION
**Pull request by robertmin1**
Add CopyObject Function #128

### CODE REVIEW

**Comment by robertmin1** [comment]
Accidentally closed the PR when force pushing :(

**Comment by robertmin1** [comment]
The error from Go Lint New : " error returned from external package is unwrapped "  I think  importing "fmt" and adding it to the return statements might fix the issue. (fmt.Error)
But some of the other implemented function have the same issue :https://github.com/namecoin/pkcs11mod/pull/142/files

File: p11mod/p11mod.go [hunk 1]
```diff
@@ -410,9 +410,35 @@ func (ll *llBackend) CreateObject(sh pkcs11.SessionHandle, temp []*pkcs11.Attrib
...
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return 0, err
+	}
+
+	// Handles are 1-indexed, while our slice is 0-indexed.
+	objectIndex := int(o - 1)
+
+	if objectIndex < 0 || objectIndex >= len(session.objects) {
+		log.Printf("p11mod CopyObject: index invalid: requested %d, object count %d\n", objectIndex, len(session.objects))
```
Comment (JeremyRand) [inline p11mod/p11mod.go:422]: Can you change "index invalid" to "object index invalid" for maximum clarity?

**Comment by JeremyRand** [comment]
I'm not going to worry about the `wrapcheck` warning.  One last small review item above, then ACK 20b42b083ad3fdf60f2aabea4f8b1c8baacd9102.

File: p11mod/p11mod.go (hunk 1 above)
Comment (robertmin1) [reply p11mod/p11mod.go]: Okay!

Comment (robertmin1) [reply p11mod/p11mod.go]: Everything looks good!

**Comment by robertmin1** [comment]
Added the extra word for maximum clarity

**Comment by JeremyRand** [comment]
ACK 2ab6dde001c329436d821d08f120bc36a76f15bb

**Pull request merged**